Sources/SourceDocs/Classes/SwiftDocDictionary.swift

```diff
@@ -16,8 +16,8 @@ extension Dictionary where Key == String, Value == Any {
         return self[key.rawValue] as? T
     }
 
-    var hasPublicACL: Bool {
-        return self["key.accessibility"] as? String == "source.lang.swift.accessibility.public"
+    var ACL: AccessLevel {
+        return AccessLevel(rawValue: self["key.accessibility"] as? String ?? "source.lang.swift.accessibility.public")!
```

Comment:
Please rename `ACL` to `accessLevelControl`, `accessLevel`, or `acl`. Properties shouldn't be uppercased.

In addition, please remove the forced-unwrap, and default to `private` ACL if no access level is available.

Reply:
Actually, I'll take care of these once landed, thank you!